tests/transforms/test_random_link_split.py

```diff
@@ -0,0 +1,19 @@
+import os
+import os.path as osp
+import sys
+import warnings
+warnings.filterwarnings('ignore')
+os.environ['TL_BACKEND']='torch'
+sys.path.append(os.path.abspath('.'))
+
+import tensorlayerx as tlx
+tlx.set_device()
+
+from gammagl.datasets import Planetoid
```

Comment:
wrap it with a func `def test_random_link_split():`.
Ref to https://github.com/pyg-team/pytorch_geometric/blob/a6e349621d4caf8b381fe58f8e57109b2d0947ed/test/transforms/test_random_link_split.py